bin_props:
  _1:
    name: bin_1_part_64621
    part_id: 64621

part_props:
  _4:
    name: Geared motor
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: tray_1_partition_4
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.06]
    speed_slow: 0.1
    use_normals: true
  _5:
    name: Pully for round belt
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: tray_2_partition_6
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.03]
    speed_slow: 1.0
    use_normals: true
  _6:
    name: Polyurethane round belt
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: tray_1_partition_3
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, 0.0]
    place_offset: [0.0, 0.0, 0.05]
    speed_slow: 1.0
    use_normals: false
  _7:
    name: Bearing with housing
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: tray_1_partition_2
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.05]
    speed_slow: 0.1
    use_normals: true
  _8:
    name: Drive shaft
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: tray_2_partition_1
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.03]
    speed_slow: 0.1
    use_normals: true
  _9:
    name: End cap for shaft
    robot_name: a_bot
    camera_name: a_phoxi_m_camera
    destination: tray_2_partition_4
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.002]
    place_offset: [0.0, 0.0, 0.05]
    speed_slow: 1.0
    use_normals: false
  _10:
    name: Bearing spacers for inner ring
    robot_name: a_bot
    camera_name: a_phoxi_m_camera
    destination: tray_2_partition_7
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.002]
    place_offset: [0.0, 0.0, 0.05]
    speed_slow: 1.0
    use_normals: false
  _11:
    name: Pully for round belts clamping
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: tray_1_partition_1
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, 0,0]
    place_offset: [0.0, 0.0, 0.05]
    speed_slow: 0.04
    use_normals: true
  _12:
    name: Bearing spacer for inner ring
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: tray_2_partition_3
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, 0.0]
    place_offset: [0.0, 0.0, 0.05]
    speed_slow: 0.04
    use_normals: false
  _13:
    name: Idler for round belt
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: tray_1_partition_5
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, 0.0]
    place_offset: [0.0, 0.0, 0.05]
    speed_slow: 0.04
    use_normals: true
  _14:
    name: Bearing shaft screw
    robot_name: a_bot
    camera_name: a_phoxi_m_camera
    destination: tray_2_partition_2
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.002]
    place_offset: [0.0, 0.0, 0.05]
    speed_slow: 1.0
    use_normals: false
  _15:
    name: M6 hex nut
    robot_name: a_bot
    camera_name: a_phoxi_m_camera
    destination: tray_2_partition_5
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.002]
    place_offset: [0.0, 0.0, 0.05]
    speed_slow: 1.0
    use_normals: false
  _16:
    name: M6 flat washer
    robot_name: a_bot
    camera_name: a_phoxi_m_camera
    destination: tray_2_partition_8
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.002]
    place_offset: [0.0, 0.0, 0.05]
    speed_slow: 1.0
    use_normals: false
  _17:
    name: M4 head cap screw
    robot_name: a_bot
    camera_name: a_phoxi_m_camera
    destination: skrewholder_1
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.002]
    place_offset: [0.0, 0.0, 0.05]
    speed_slow: 1.0
    use_normals: false
  _18:
    name: M3 head cap screw
    robot_name: a_bot
    camera_name: a_phoxi_m_camera
    destination: skrewholder_2
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.002]
    place_offset: [0.0, 0.0, 0.05]
    speed_slow: 1.0
    use_normals: false
  _58331:
    name: toyota1 9R
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: workspace_center
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.06]
    speed_slow: 1.0
    use_normals: true
  _58332:
    name: toyota1 9L
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: workspace_center
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.06]
    speed_slow: 1.0
    use_normals: true
  _61653:
    name: toyota1 5R
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: workspace_center
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.06]
    speed_slow: 1.0
    use_normals: true
  _61654:
    name: toyota1 5L
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: workspace_center
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.06]
    speed_slow: 1.0
    use_normals: true
  _64621:
    name: toyota1 9ZB
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: workspace_center
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.06]
    speed_slow: 1.0
    use_normals: true
  _68945:
    name: toyota1 WR
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: workspace_center
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.06]
    speed_slow: 1.0
    use_normals: true
  _68946:
    name: toyota1 WL
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: workspace_center
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.06]
    speed_slow: 1.0
    use_normals: true
  _69430:
    name: toyota1 Lock
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: workspace_center
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.06]
    speed_slow: 1.0
    use_normals: true
  _89997:
    name: toyota1 9L18
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: workspace_center
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.06]
    speed_slow: 1.0
    use_normals: true
  _64600:
    name: toyota1 Trunk
    robot_name: b_bot
    camera_name: a_phoxi_m_camera
    destination: workspace_center
    approach_offset: [0.0, 0.0, 0.15]
    grasp_offset: [0.0, 0.0, -0.003]
    place_offset: [0.0, 0.0, 0.06]
    speed_slow: 1.0
    use_normals: true
